hbase-client/src/main/java/org/apache/hadoop/hbase/client/CompactType.java

```diff
@@ -26,9 +26,7 @@
 @InterfaceAudience.Public
 public enum CompactType {
 
-  NORMAL(0),
-  MOB(1);
+  NORMAL,
```

Comment:
Correct, the value passed to the constructor was not exposed. Someone might wonder if CompactType#ordinal would return these special values but it would not.